client/src/locales/ro/messages.json

```diff
@@ -217,12 +217,12 @@
     ],
     "extractedComments": []
   },
-  "Check here if a building is on the <0><1>seismic risk</1><2/></0> list": {
-    "translation": "Verifică aici dacă o clădire se află pe lista de <0><1>risc seismic</1><2/></0>",
+  "Check here if a building is on the <0><1><2>seismic risk</2><3/></1></0> list": {
+    "translation": "",
```

Comment:
It seems the Romanian translation was removed here. Everything should be ready to merge once you add it back 😁

Reply:
done